src/module-vsbridge-indexer-catalog/Model/ResourceModel/Product/Configurable.php

```diff
@@ -267,7 +267,7 @@ private function getConfigurableAttributeFullInfo()
                         $this->configurableAttributesInfo[$attributeId] = [
                             'attribute_id' => (int)$attributeId,
                             'attribute_code' => $attributeModel->getAttributeCode(),
-                            'label' => $attributeModel->getStoreLabel(),
+                            'label' => $attributeModel->getStoreLabel($storeId),
```

Comment:
we should add internal caching by attribute id and store, 
```
$cacheKey = $attributeId . '_' . $storeId;
..
if ($this->configurableAttributesInfo[$cacheKey])
```

or just make separated caching for attribute store labels - it would probably be best, because we only used this store labels in `getProductConfigurableAttributes` I think. 

If we reindex one store (using vsbridge:reindex command) there won't be a problem, but magento indexers reindex all stores.. so we might got some wrong labels.